src/marketing/magento-product-reviews-moderate.md

```diff
@@ -17,13 +19,30 @@ For Commerce Product Reviews, a submitted product review must be approved before
    {:.bs-callout-info}
    Reviews with the `Pending` and `Not Approved` statuses are not displayed on the storefront.
 
+1. If applicable, set the **Visibility** of a product review for appearing in different store views.
```

Comment:
"Visibility" is not in the screen shot. Does it need a refresh?

Reply:
@dobooth , thank you for noticing.

"Visibility" is appeared only if there are a few store views f.e. That is why I have added "If applicable"

Seems like on the provided screenshot, there is only one website/store/store view.

Of course we can edit an image, where the "Visibility" field will be present.

For example this one.

<img width="2458" alt="Screenshot 2022-01-14 at 16 39 15" src="https://user-images.githubusercontent.com/40993770/149533938-ab39f2fa-8279-4528-84d5-d3dde29bed2e.png">


Thanks!